app/i18n/locales/en.json

```diff
@@ -54,6 +54,11 @@
   "Are_you_sure_question_mark": "Are you sure?",
   "Are_you_sure_you_want_to_delete_your_account": "Are you sure you want to delete your account?",
   "Are_you_sure_you_want_to_leave_the_room": "Are you sure you want to leave the room {{room}}?",
+  "At_Least_1_Lowercase_Letter": "At least 1 lowercase letter",
+  "At_Least_1_Number": "At least 1 number",
+  "At_Least_1_Symbol": "At least 1 symbol",
+  "At_Least_8_Characters": "At least 8 characters",
+  "At_Most_24_Characters": "At most 24 characters",
```

Comment:
![image](https://github.com/user-attachments/assets/71f57f9d-7911-4e15-b727-047b9ccb4c68)

Shouldn't be fixed.

Reply:
got it, updated.